_config.yml

```diff
@@ -1,6 +1,6 @@
 title: Data Patterns
-short_title: Data Patterns
-description: "A collection of tips, tricks and patterns for data work."
+short_title: Labs Handbook
+description: "A collection of guides, counselling and other advise to participate in OKFN Labs"
```

Comment:
"counselling" is odd phrasing. I would just have:

"Guides and advice on participating in Open Knowledge Labs and its projects"